Dockerfile.ubi8

```diff
@@ -0,0 +1,21 @@
+# Build the manager binary
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 as builder
```

Comment:
all KServe images use golang 1.21, as it is the builder only, you can use golang:1.21 instead gotoolset.
same for the other Dockerfiles.

Reply:
go-toolset is ok to use. For golang 1.21, we should use ubi9. There are some discussions about UBI9 transition but I think this is ok because it is builder image, actual image is still UBI8